src/main/java/hng_java_boilerplate/sale/service/SaleService.java

```diff
@@ -0,0 +1,8 @@
+package hng_java_boilerplate.sale.service;
+
+public interface SaleService {
+    Object getSummary();
```

Comment:
Specify the return type

Reply:
I left it like that because i don't have to explicitly declare public to the methods in an interface, thats redundant

Reply:
Return type not access modifier